services/utils/classNames.js

```diff
@@ -0,0 +1,3 @@
+export function classNames(...classes) {
+    return classes.filter(Boolean).join(" ");
```

Comment:
Please check your Prettier settings in VScode, the indentation doesn't look quite right

Reply:
ok will keep in mind from next time